Fixed some bugs those addressed on issue pages and added `int1` alias as domain.

krmgns opened the pull request:
Fixes:
1- ERROR: could not find function information for function "bt_tinyint_i8_cmp" [#2]
2- No longer able to build on Postgresql 11, Ubuntu 18.04 [#4]

krmgns committed:
Fixed some bugs those addressed on issue pages and added `int1` alias as domain.

krmgns committed:
Update README about index issues.

krmgns committed:
Update README (table).

krmgns committed:
Update README (table).

umitanuki commented:
Thank you for PR! I am currently not maintaining the module actively, but I wish someone can take this over...

krmgns committed:
Doc updates.

k-gun commented:
@umitanuki Hi there, I think I can do it.

k-gun commented:
> Thank you for PR! I am currently not maintaining the module actively, but I wish someone can take this over...

Hi there, I think I can do it.

krmgns commented:
> Thank you for PR! I am currently not maintaining the module actively, but I wish someone can take this over...

Hi there, I think I can do it.

krmgns committed:
Doc updates.

krmgns committed:
Doc updates.

krmgns committed:
Doc updates.

krmgns committed:
Doc updates.

krmgns committed:
Doc updates.

krmgns committed:
Fix sh commands.

Dead2 commented on Makefile:
```diff
@@ -1,31 +1,31 @@
+EXTRA_CLEAN += $(SQL_IN) $(EXTSQL) $(EXTSQLTMP)
 
-USE_EXTENSION = $(shell $(pg_config) --version | grep -qE " 8\.|9\.0" && echo no || echo yes)
+USE_EXTENSION = $(shell pg_config --version | grep -qE " 8\.|9\.0" && echo no || echo yes)
```
This does a partial revert of the changes in c47c39e0af36ee85e14969b58700bc3d34f16978, making it harder to compile this module for a non-default postgresql version.
Such as compiling for pgsql 10/11/12 from pgdg repos when the distro only provides postgresql 9.6 (Such as is the case for EL/Centos 7).

krmgns replied:
Sadly it did not work on Ubuntu and could not figure out in other way. I thing there need to be more if/else conditions.

krmgns committed:
Username updates.

mkgrgis commented:
Any activity here?

mkgrgis committed:
Documentation refactoring

Based on structure of https://github.com/ibarwick/firebird_fdw/blob/master/README.md

krmgns committed:
Merge pull request #1 from mkgrgis/patch-1

Documentation refactoring

mkgrgis committed:
Fix typo, add link to other realisation of 1b int

krmgns committed:
Merge pull request #2 from mkgrgis/patch-1

Fix typo, add link to other implementation of 1b int

krmgns committed:
Update README.md
